src/wp-admin/network/users.php

```diff
@@ -107,12 +118,27 @@
 
 							case 'notspam':
 								$user = get_userdata( $user_id );
+								if ( is_super_admin( $user->ID ) ) {
+									wp_die(
+										sprintf(
+											/* translators: %s: User login. */
+											__( 'The user %s is a network administrator and cannot be modified.' ),
+											esc_html( $user->user_login )
+										)
```

Comment:
As per Core-63836, we shouldn't use the default error code of `500`. A `403 Forbidden` status would seem to make sense.

```suggestion
										),
										403
```

Reply:
OK, thanks for the clarification!